test_regress/t/t_case_auto1.pl

```diff
@@ -11,6 +11,7 @@
 scenarios(simulator => 1);
 
 compile(
+    v_flags => ['--x-initial-edge']
```

Comment:
Why do we need this now? Note: with my scheduling changes this will probably go away so no need to dig deep, but it is suspect.